fix: licensing policy and arn info (#4004)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/licensing/add/index.md

beta: false
---

## Prerequisites

> For licenses purchased via the [AWS Marketplace](https://aws.amazon.com/marketplace/), an AWS administrator must attach the [AWS managed policy](https://docs.aws.amazon.com/license-manager/latest/userguide/security-iam-awsmanpol.html#security-iam-AWSLicenseManagerConsumptionPolicy) `AWSLicenseManagerConsumptionPolicy` to the `AWSIAMRoleControlPlane` role created when [configuring AWS IAM policies for Konvoy](/dkp/konvoy/latest/choose-infrastructure/aws/iam-policies/).

Reviewer: ```suggestion
For licenses purchased via the [AWS Marketplace](https://aws.amazon.com/marketplace/), an AWS administrator must attach the [AWS managed policy](https://docs.aws.amazon.com/license-manager/latest/userguide/security-iam-awsmanpol.html#security-iam-AWSLicenseManagerConsumptionPolicy) `AWSLicenseManagerConsumptionPolicy` to the `control-plane.cluster-api-provider-aws.sigs.k8s.io` role created when [configuring AWS IAM policies for Konvoy](/dkp/konvoy/latest/choose-infrastructure/aws/iam-policies/).
```

Without this policy attached to the role, Kommander cannot verify the license information provided in the steps below.

## Obtain a license token or AWS license ARN

For licenses purchased directly from D2iQ, obtain the license token via the customer support portal. Input this token in the last step below.

For licenses purchased via the AWS marketplace, find the license in the "Granted Licenses" table of AWS License Manager inside the AWS console. If necessary, modify the table view preferences to show "License ARN." Copy this value for use in the last step below.

## Enter license information

> An administrator must add licenses to Kommander.

Reviewer 2: ```suggestion
<p class="message--note"><strong>NOTE: </strong>An administrator must add licenses to Kommander.</p>
```

> In the Kommander UI, do the following:

Reviewer 2: ```suggestion
From the Kommander UI, complete the following steps:
```
